linode/manager
# refactor: [M3-6377] - MUI v5 - Components > ProductInformationBanner

## Description 📝
**MUI migration SRC > Components > ProductInformationBanner**

## Major Changes 🔄

- Run tss-react codemod on `ProductInformationBanner` 
- Renamed Props to    ProductInformationBannerProps.
- Changes default exports to named exports.

## How to test 🧪

- Verify all the references of `ProductInformationBanner`.
- Verify e2e tests pass
